hols/todo/infra/multi-cluster/scripts/setup.template.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+# Copyright (c) 2025 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+for f in cilium coredns database istio todo; do
```

Comment:
Where is this used ? Can't this be inlined ?

Reply:
terraform creates the directories on the operator host and then upload scripts and manifests into them

Reply:
IIUC it's executed with this:
```
  provisioner "remote-exec" {
    inline = [
      "mkdir /home/opc/bin; mv token_helper.sh /home/opc/bin; chmod +x /home/opc/bin/token_helper.sh",
      "if [ -f \"$HOME/setup.sh\" ]; then bash \"$HOME/setup.sh\";fi",
      "if [ -f \"$HOME/install_istioctl.sh\" ]; then bash \"$HOME/install_istioctl.sh\";fi",
      "if [ -f \"$HOME/install_cilium_cli.sh\" ]; then bash \"$HOME/install_cilium_cli.sh\";fi",
    ]
  }
```

Can we combine all of those scripts into one ? Do we need the modularity ?

Reply:
maybe but it's easier to separate and maintain them.

Reply:
I could argue for days on this topic :)

Having multiple tiny scripts that are scattered and coordinated in an inline terraform statement is complicated.
Extra complexity like this gets in the way of a reader/user trying to understand the point of this exercise.

It's actually easier to maintain a single install/setup script because it's self-contained.

Overall, less files is better.

The same thing could be said about all the other convenience scripts, you could combine them all into a `todoctl` and implement sub commands for each of the use-cases.

Easier to maintain, easier to use.
You could install in the PATH, and simply tell users to do `todoctl foobar` or `todoctl --help`.

Reply:
> Where is this used ? Can't this be inlined ?

I vaguely remember trying and something wasn't quite working and I couldn't quite figure out what

Reply:
> I could argue for days on this topic :)

I have the feeling you're quite enjoying this :)